console/src/main/java/org/nervos/ckb/example/transaction/CollectUtils.java

```diff
@@ -51,17 +58,28 @@ public CollectUtils(Api api) {
 
   public List<CellOutput> generateOutputs(
       List<Receiver> receivers, String changeAddress, BigInteger fee) throws IOException {
+    return generateOutputs(receivers, changeAddress, fee, CodeHashType.BLAKE160);
+  }
+
+  public List<CellOutput> generateOutputs(
+      List<Receiver> receivers, String changeAddress, BigInteger fee, CodeHashType codeHashType)
+      throws IOException {
     if (fee.compareTo(BigInteger.ZERO) < 0) {
       throw new IOException("Transaction fee should not be smaller than zero");
     }
     List<CellOutput> cellOutputs = new ArrayList<>();
-    AddressUtils addressUtils = new AddressUtils(Network.TESTNET);
+    AddressUtils addressUtils = new AddressUtils(Network.TESTNET, codeHashType);
```

Comment:
I bet Huobi will want to use mainnet addresses as well, should we make this tunable?

Reply:
I see. This is just a demo for huobi(in actually they also just need a demo), and using testnet address can be run in ckb testnet blockchain.

Reply:
Okay I found out my mistake: I only look at the file name `CollectUtils.java` but didn't see the `example` part in the package name, I thought this is actual SDK code, in this case never mind.

Reply:
OK. Thanks. I will also make sure mainnet address is ok in my local computer.

Reply:
I tried with mainnet address in my local computer and it did work well.